electron/src/electron/network.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  WebPlugin, NetworkPlugin, NetworkStatus
+} from "@capacitor/core";
+
+export interface PluginListenerHandle {
+  remove: () => void;
+}
+
+declare var window:any;
+
+export class NetworkPluginElectron extends WebPlugin implements NetworkPlugin {
+
+  listenerFunction:any = null;
+
+  constructor() {
+    super({
+      name: 'Network',
+      platforms: ['electron']
+    });
+  }
+
+  getStatus(): Promise<NetworkStatus> {
+    return new Promise((resolve, reject) => {
+      if(!window.navigator)
+        reject('Network info not available');
+      let connected = window.navigator.onLine;
+      resolve({connected: connected, connectionType: connected ? 'unknown' : 'none'});
+    });
+  }
+
+  addListener(eventName: 'networkStatusChange', listenerFunc: (status: NetworkStatus) => void): PluginListenerHandle {
+    let thisRef = this;
+    let onlineBindFunc = listenerFunc.bind(thisRef,{connected: true, connectionType: 'unknown'});
```

Comment:
Same comment: can we use connection type? https://developer.mozilla.org/en-US/docs/Web/API/Network_Information_API